x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -301,3 +301,46 @@ export function databaseDocumentIDToAbort(state: DataState): string | null {
     return null;
   }
 }
+
+/**
+ * `ResolverNodeStats` for a process (`ResolverEvent`)
+ */
+const relatedEventStatsForProcess: (
+  state: DataState
+) => (event: ResolverEvent) => ResolverNodeStats | null = createSelector(
+  relatedEventsStats,
+  (statsMap) => {
+    if (!statsMap) {
+      return () => null;
+    }
+    return (event: ResolverEvent) => {
+      const nodeStats = statsMap.get(uniquePidForProcess(event));
```

Comment:
Just leaving the note here like we talked about before @oatkiller this function `uniquePidForProcess` will suffer from the same problem we talked about before where `unique_pid` will have collisions in the map just like the `entityId` function. We'll need to do the serialization or something for that function too.

Reply:
I'll get something formally added to the team's backlog about this.